Pocket/pocket-ios
# feat(syndication): add indicator for syndicated articles

## Summary
Add icon indicator for syndicated articles, which is represented via a checkmark.

## References 
IN-1580

## Implementation Details
* Created `addSyndicatedIndicator` function to be able to apply to any `NSMutableAttributedString` in which it adds the syndicated icon to the text.
* The image was not centering with the text, so created a private method `calculateLineHeight` based on this stack overflow post: https://stackoverflow.com/questions/26105803/center-nstextattachment-image-next-to-single-line-uilabel

## Test Steps
1. Find a syndicated article (i.e. https://getpocket.com/explore/item/to-find-success-at-work-match-your-job-with-your-personality?utm_source=pocket_reader)
2. Save it to your list
3. Verify that the syndicated icon appears in both Home and Saves tab

## PR Checklist:
- N/A Added Unit / UI tests - no tests added for Textile files so did not spend engineering effort due to the transitioning

## Screenshots
![image](https://github.com/Pocket/pocket-ios/assets/6743397/d9b3f65b-b67c-445f-a9c1-c42e3175ae68)

| iPad Home | iPad Saves | 
| --- | --- |
| ![image](https://github.com/Pocket/pocket-ios/assets/6743397/1287be9d-34db-452a-8aac-38780e06cced) |  ![image](https://github.com/Pocket/pocket-ios/assets/6743397/1f9f5628-a7a2-4e37-8cfd-4b8209db4de8) |
